orchardcore.ruleset

```diff
@@ -2,15 +2,20 @@
 <RuleSet Name="Orchard Core C# rules" ToolsVersion="17.0">
   <Include Path="general.ruleset" Action="Default" />
   <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
+    <Rule Id="CA1014" Action="None" />
     <Rule Id="CA1710" Action="Hidden" />
     <Rule Id="CA1711" Action="Hidden" />
     <Rule Id="CA1716" Action="Hidden" />
   </Rules>
+  <Rules AnalyzerId="Microsoft.CodeAnalysis.CSharp" RuleNamespace="Microsoft.CodeAnalysis.CSharp">
+    <Rule Id="CS1591" Action="None" />
```

Comment:
I would love for these R# warnings to disappear, but they're still there:

![image](https://user-images.githubusercontent.com/164360/171232293-8461da57-2cd3-434b-a425-13a8b977e324.png)

Any idea?

Reply:
I found a solution [here](https://github.com/Lombiq/.NET-Analyzers/pull/62/files#diff-0947e2727d6bad8cd0ac4122f5314bb5b04e337393075bc4b5ef143b17fcbd5bR133).